cmd/sink/main.go

```diff
@@ -19,7 +24,17 @@ func receive(event cloudevents.Event) {
 }
 
 func main() {
-	client, err := cloudevents.NewClientHTTP()
+	err := observability.Start()
+	if err != nil {
+		logger.Printf("Could not start tracing: %s\n", err.Error())
+	}
+	httpClient, err := cloudevents.NewHTTP(
+		cloudevents.WithRoundTripper(otelhttp.NewTransport(http.DefaultTransport)),
+		cloudevents.WithMiddleware(func(next http.Handler) http.Handler {
+			return otelhttp.NewHandler(next, "receive")
+		}),
+	)
+	client, err := cloudevents.NewClient(httpClient, ceClient.WithObservabilityService(otelObs.NewOTelObservabilityService()))
```

Comment:
Since you have added an "httpClient" variable to this method, having a "client" variable is a bit confiusing. Can you update it to something a bit more descriptive, like "ceClient"?

Reply:
I'm already using `ceClient` as the name for the cloudEvents client module. I renamed `client` to `eventClient` instead